src/utils.jl

```diff
@@ -96,6 +102,22 @@ function classify(path::OrbitPath, axis)
     return classify(path.r,path.z,path.pitch, axis, n=length(path))
 end
 
+function class_char(class::Symbol)
```

Comment:
Could you provide some documentation for the class_char() functions? @StuartBenjamin

Reply:
Hi Henrik I'm just adding this now

Reply:
Great, thanks!